src/MPoly.jl

```diff
@@ -425,16 +425,27 @@ end
 
 iszero(x::MPolyRingElem{T}) where T <: RingElement = length(x) == 0
 
-function is_unit(a::MPolyRingElem{T}) where T <: RingElement
-   if is_constant(a)
-      return is_unit(leading_coefficient(a))
-   elseif is_domain_type(elem_type(coefficient_ring(a)))
+# function is_unit(a::MPolyRingElem{T}) where T <: RingElement
+#    if is_constant(a)
+#       return is_unit(leading_coefficient(a))
+#    elseif is_domain_type(elem_type(coefficient_ring(a)))
+#       return false
+#    elseif length(a) == 1
+#       return false
+#    else
+#       throw(NotImplementedError(:is_unit, a))
+#    end
+# end
```

Comment:
```suggestion
```

Reply:
Thanks -- i plan to delete commented out code when the tests pass... if ever!